scripts/deploy.ts

```diff
@@ -0,0 +1,54 @@
+import { Wallet } from 'ethers';
+
+import {
+  Chains,
+  HyperlaneCore,
+  MultiProvider,
+  objMap,
+} from '@hyperlane-xyz/sdk';
+import { RouterConfig, chainConnectionConfigs } from '@hyperlane-xyz/sdk';
+
+import { CollateralConfig, SyntheticConfig } from '../src/config';
+import { HypERC721Deployer } from '../src/deploy';
+
+const connectionConfigs = {
+  celo: chainConnectionConfigs.celo,
+  ethereum: chainConnectionConfigs.ethereum,
+};
+
+async function deployNFTWrapper() {
+  console.info('Getting signer');
+  const signer = new Wallet(
+    'pkey',
+  );
+
+  const multiProvider = new MultiProvider(connectionConfigs);
+  multiProvider.rotateSigner(signer)
+  const core = HyperlaneCore.fromEnvironment('mainnet', multiProvider);
+  const config = {
+    celo: {
+      type: 'SYNTHETIC_URI',
```

Comment:
Should this be just SYNTHETIC to match the const in config?
https://github.com/hyperlane-xyz/hyperlane-token/blob/main/src/config.ts#L6

Also, would be more conventional to have these be enums, whose values would typically be lowercase. cc @yorhodes